x-pack/plugins/lists/common/schemas/types/entries.ts

```diff
@@ -12,12 +12,26 @@ import { entriesMatch } from './entry_match';
 import { entriesExists } from './entry_exists';
 import { entriesList } from './entry_list';
 import { entriesNested } from './entry_nested';
+import { entriesMatchWildcard } from './entry_match_wildcard';
 
-export const entry = t.union([entriesMatch, entriesMatchAny, entriesList, entriesExists]);
+export const entry = t.union([
```

Comment:
Just thought maybe worth adding a note here like:
```suggestion
// NOTE: Type nested is not included here to denote it's non-recursive nature. So a nested entry is really just a
// collection of `Entry` types.
export const entry = t.union([
```

Reply:
done ddc3d72c805ed7f25fe7f2184bb98d51ca8426d6